src/components/fields/edit/LocationIdEditor.jsx

```diff
@@ -94,9 +99,9 @@ export default function LocationIdEditor(props) {
           }
         }}
         getOptionLabel={option => get(option, 'name', '')}
-        getOptionSelected={(option, val) => {
-          return option.id ? option.id === val.id : false;
-        }}
+        getOptionSelected={option =>
```

Comment:
Why make this independent of val? I don't understand what benefit is gained by doing it this way.

Reply:
Same difference, I think. I just changed it as I was debugging and never changed it back since it was working well